### ISSUE TITLE
Failure on vagrant up: Inappropriate ioctl for device

### ISSUE THREAD

**Issue by jeffhipp** [geerlingguy/drupal-vm#659]
I keep getting this failure whenever I run `vagrant up`, including during the initial spin-up.  My system is OSX 10.11.x.

I get this error when using either my customized config.yml,  or the example config file included in the package.

This issue seems particular to drupal-vm -- I've spun up other Vagrant boxes today and haven't had the same problem.

Any guidance? Thanks!

```
=> drupalvm-3: Running provisioner: shell... 
    drupalvm-3: Running: /var/folders/j6/5nnfqgtn73s68qnxj1r1g9mw0000gp/T/vagrant-shell20160524-3245-1a3d5uc.sh 
==> drupalvm-3: mesg:  
==> drupalvm-3: ttyname failed 
==> drupalvm-3: :  
==> drupalvm-3: Inappropriate ioctl for device
```

**Comment by oxyc** [comment]
Are you using Vagrant 1.8.1?

**Comment by jeffhipp** [comment]
Yes, Vagrant 1.81, and `geerlingguy/ubuntu1604  = (virtualbox, 1.0.1)`

**Comment by geerlingguy** [comment]
@jeffhipp - Can you `vagrant destroy` and then `vagrant up` again? Maybe it has an older version of the box, or there was another box being used... that generally only happens if the base box doesn't have networking configured properly (e.g. https://github.com/mitchellh/vagrant/issues/6871)

**Comment by geerlingguy** [comment]
Also, on Windows, Mac or Linux?

**Comment by jeffhipp** [comment]
@geerlingguy ,

>  Can you vagrant destroy and then vagrant up again?

 I've already tried that a few times, unfortunately -- still fails.

> Also, on Windows, Mac or Linux?

Mac OSX 10.11.

Thanks!

**Comment by jeffhipp** [comment]
Closing the loop here -- a friend with more vagrant experience actually identified a different cause of the critical failure, further down in the log -- we addressed the issue and drupal-vm is now working properly for me, despite the ioctl message still displaying.

However, my friend realized that he was also receiving the `Inappropriate ioctl for device` message during the initial spin-up on his Mac, though the box functions properly. Not sure how much that matters, but I wanted to be sure to note it in this thread.

Thanks again!
